src/pkg/projects/Microsoft.NETCore.App/src/Microsoft.NETCore.App.depproj

```diff
@@ -152,6 +146,13 @@
       <_sourcePath Include="@(_sourcePathCandidate)" Condition="Exists('%(Identity)')" />
     </ItemGroup>
 
+    <!-- Add versions file with the hashes of the repos we consume -->
+    <ItemGroup>
+      <FilesToPackage Include="$(IntermediateOutputPath)\Microsoft.NETCore.App.versions.txt">
+        <TargetPath></TargetPath>
```

Comment:
This same target [is basically now getting all of the elements that currently exist in `FilesToPackage` and setting the TargetPath](https://github.com/dotnet/core-setup/blob/master/src/pkg/projects/Microsoft.NETCore.App/src/Microsoft.NETCore.App.depproj#L140-L155) to that runtimes path whenever RuntimeID is not empty. Given that the way I was adding this text file was staticly (outside a target), then it was always being added before this task ran which now started setting the targetPath for it. With this change, I'm adding the file to the Item AFTER the TargetPath is set, therefore it won't override my empty one anymore.